source/extensions/access_loggers/fluentd/fluentd_access_log_impl.cc

```diff
@@ -140,7 +140,7 @@ FluentdAccessLog::FluentdAccessLog(AccessLog::FilterPtr&& filter, FluentdFormatt
                                    ThreadLocal::SlotAllocator& tls,
                                    FluentdAccessLoggerCacheSharedPtr access_logger_cache)
     : ImplBase(std::move(filter)), formatter_(std::move(formatter)), tls_slot_(tls.allocateSlot()),
-      config_(config), access_logger_cache_(std::move(access_logger_cache)) {
+      config_(config), access_logger_cache_(access_logger_cache) {
```

Comment:
I think you needn't the `access_logger_cache_` member now?

Reply:
I'm not sure, because I need it to call on ``getOrCreateLogger`` within the lambda, if I don't save it as a member the ``access_logger_cache`` is not valid within the lambda anymore

Reply:
You can capture the cache in the lambda directly and needn't store it as a member.